app/src/main/res/values/strings.xml

```diff
@@ -655,4 +655,8 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="label_housenumber">house number</string>
     <string name="label_blocknumber">block number</string>
     <string name="quest_busStopShelter_covered">Whole stop is covered</string>
+    <string name="quest_ferry_pedestrian_name_title">Is %s accessible to pedestrians?</string>
+    <string name="quest_ferry_pedestrian_title">Is this ferry accessible to pedestrians?</string>
+    <string name="quest_ferry_motor_vehicle_name_title">Is %s accessible to motor vehicles?</string>
+    <string name="quest_ferry_motor_vehicle_title">Is this ferry accessible to motor vehicles?</string>
```

Comment:
Not sure if "accessible" is the right word for it. Rather "does X transport...?" or "is X for ...?" or something like that?

Reply:
I like "does X transport...?" (it will fail for private ferries without public access but that should be ridiculously rare and I expect original mapper to tag it explicitly)

Reply:
I suggest "Are __ permitted to ride on %s?"

Reply:
Looking at POEditor (I was translating latest strings to Polish) I noticed that while

> Does %FERRY_NAME transport pedestrians?

is OK, but variants for nameless ferries

> Does transport pedestrians?

and

> Does transport motor vehicles?

seem broken. It should be probably

> Does ferry transport pedestrians?

and

> Does ferry transport motor vehicles?

Reply:
*this ferry route

Reply:
I updated Engrish-fixing PR to use this form.